sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserRolesView.java

```diff
@@ -120,7 +121,7 @@ public HorizontalLayout createStatusFilterBar() {
 		enabledFilter.setId(UserRoleDto.ENABLED);
 		enabledFilter.setWidth(200, Unit.PIXELS);
 		enabledFilter.setInputPrompt(I18nProperties.getPrefixCaption(UserRoleDto.I18N_PREFIX, UserRoleDto.ENABLED));
-		enabledFilter.addItems(ENABLED_FILTER, DISABLED_FILTER);
+		enabledFilter.addItems(ALL_FILTER, ENABLED_FILTER, DISABLED_FILTER);
```

Comment:
instead of adding a new item could you explore if we could just add a "Boolean.TRUE" instead of null in criteria enabled logic?